packages/iov-core/src/index.ts

```diff
@@ -16,4 +16,4 @@ export {
 } from "@iov/keycontrol";
 export { ChainId } from "@iov/tendermint-types";
 
-export { bnsConnector, ChainConnector, IovWriter } from "./writer";
+export { IovWriter } from "./writer";
```

Comment:
We no longer export ChainConnector anywhere?
I guess we only want concrete implementations, but if someone imports iov-core from npm to add support for another blockchain, they may want that type.

Reply:
> if someone imports iov-core from npm to add support for another blockchain, they may want that type.

yes, they get the type from @iov/bcp-types, which now includes everything you need to create a custom connector. As you can see in https://github.com/iov-one/iov-core/pull/376/commits/ed1338c1e9349cf2a82b41a97235a931c5693e32, @iov/lisk gets rid of the @iov/core dependency by this change.

Reply:
cool. :+1: